src/hypermedia-constants.js

```diff
@@ -508,6 +508,9 @@ export const Actions = {
 		updateExternalResource: 'update-external-resource',
 		deleteWeblink: 'delete-webLink',
 		deleteLTIlink: 'delete-ltiLink',
+		checkoutWebLink: 'checkout',
```

Comment:
just gonna document this here now, definitely no need to change or even worry about it yet for HTML documents.. but this looks like a good opportunity to make this pretty generic and have some sort of `"ContentWorkingCopyEntity"` that will just automatically get these actions and we won't need to rewrite and add them for every new type

Reply:
Yeah this is a good call, and something I believe @quinnbudan suggested also. Seems like a good refactor for a future PR .